website/docs/webdev/3-Guidelines/2-colours.md

```diff
@@ -18,20 +18,20 @@ Accessibility is crucial and AA colour contrast must be achieved. It is up to th
 This color should be eye-catching but not harsh. It can be liberally applied to your layout as its main identity. A primary color is the color displayed most frequently across your screens and components.
 
 <div style={{width:'100%',float:'left'}}>
-    <table style={{float:'left'}} >
+    <table style={{float:'left'}}>
         <tbody>
             <tr>
                 <td style={{backgroundColor:'#007B6C',width:200+'px',height:150+'px',textAlign:'center'}}>
-                    <div style={{color:'white',width:100+'%',textAlign:'center',height:150+'px',display:'flex',flexDirection:'column',justifyContent:'center'}}>#007B6C <br/> EEA GREEN</div>
+                    <div style={{color:'white',width:100+'%',textAlign:'center',height:150+'px',display:'flex',flexDirection:'column',justifyContent:'center'}}>#007B6C<br/>EEA GREEN</div>
```

Comment:
@g-stamatis all of these styles could have been added in custom.css instead of having it inline copied over and over as it makes it hard to maintain and add new things when you have so much inline style.

The table could of received a class such as swatches-table where then every td has the style and the color
div could also get a class making it easier to maintain and to avoid so much inline copy and paste of css